training/distributed_training/pytorch/data_parallel/bert/pytorch_smdataparallel_bert_demo.ipynb

```diff
@@ -22,7 +22,7 @@
     "6. Define training metrics\n",
```

Comment:
<p>This cell requires user inputs. So it cannot pass CI. Would be better if there's way to use <code>sagemaker.Session()</code> to get some default inputs.</p><p> </p><p>It is not clear where the data should be saved in Fsx. Suppose users mount Fsx to their ec2 instance to <code>/fsx</code>,  then what should be the directory structure of the data in <code>/fsx</code>  so that the model can access it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>